spec/dummy_controller_spec.rb

```diff
@@ -214,5 +214,64 @@
         expect(response_body(response)['total_count']).to eq 3
       end
     end
+
+    context 'when paginating with a custom adapter' do
+      include_context 'with default pagination params'
+
+      before do
+        pagination_params[:count] = pagination_params[:page] = 8
+        pagination_params[:total_pages] = 4
```

Comment:
Why this?

Reply:
This is because the CustomAdapter that I have as an example to this, changes the default values of page, count, total_pages. So I'm changing those values in the `pagination_params` hash before the test runs, in that way, when the shared example `proper pagination params` tries to compare with the `pagination_params` hash, it will have the expected values.